stage/codefresh/addons/external-nodes/runtime-environments/example-node-1.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
consider to put it under "examples" directory

Reply:
done

Reply:
done in #15